static/src/javascripts/components/enhancer/enhancer.js

```diff
@@ -1,8 +1,4 @@
-define([
-    'require'
-], function (
-    require
-) {
+define(function () {
```

Comment:
I'm seeing really mixed results, and I'm not sure why. I expected to see a failing dynamic require, as before, but when I reverted the changes in this PR, the require actually worked. Can you confirm which version of system js supports the new spec regarding URL module names, https://github.com/systemjs/systemjs/issues/247? I'm testing with 0.16.11.